vendor/libvirt.org/libvirt-go/domain.go

```diff
@@ -3298,7 +3298,9 @@ func (d *Domain) GetJobStats(flags DomainGetJobStatsFlags) (*DomainJobInfo, erro
 	}
 	defer C.virTypedParamsFree(cparams, cnparams)
 
-	params := DomainJobInfo{}
+	params := DomainJobInfo{
```

Comment:
Posted a PR for libvirt-go
https://gitlab.com/libvirt/libvirt-go/-/merge_requests/27